BasicAssistant/BasicAssistant/Nuget.config

```diff
@@ -1,8 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I don't know if we should remove this from the sample. Microsoft policies prevent us from using multiple nuget feeds but that's not the case for our customers.
Also, this file is generated by composer templates, the goal of this sample is to deploy composer created bots so I wouldn't want to alter the code generated by composer

Reply:
FYI this is my PR removing nuget.config from the Composer bots: https://github.com/microsoft/BotFramework-Composer/pull/9094 
Nuget.config is a legacy from before the SDK went public. It is no longer useful.

Reply:
OK, is this file also present in the composer template components? (empty bot, core bot, etc.)

Can you check with Ryan if this is the case and it needs to be removed from the other packages?

![image](https://user-images.githubusercontent.com/12264946/161759891-1d885941-0ab3-4d0e-9898-acec95529c84.png)

Reply:
It is not in any of the templates. It is injected by a piece of code in Composer when a bot project is created.